src/UI/QuickmenuView.re

```diff
@@ -187,7 +187,10 @@ let make =
          | EditorsPicker => React.empty
          | _ => <input />
          }}
-        <dropdown />
+        {switch (variant) {
+         | Wildmenu(_) => React.empty
+         | _ => <dropdown />
+         }}
```

Comment:
This isn't right though. The Wildmenu has autocompletion for commands and lots of other things.

Reply:
You need to be more specific. I think this should be OK though:

```suggestion
        {switch (variant) {
         | Wildmenu(SearchForward | SearchBackward) => React.empty
         | _ => <dropdown />
         }}
```

Reply:
Nice to know that I can use `|` with variants.Thx!